### DISCUSSION TITLE
ESPHome Bluetooth Proxies?

### DISCUSSION THREAD

**Discussion by CumpsD** [mKeRix/room-assistant#1450] [Q&A]
New to room assistant, considering trying it out. Right now I have multiple ESP boards around the home acting as Bluetooth Proxies for Home Assistant.

Can I reuse this with room assistant? Or do I really need a raspberry per room?

I'd prefer using my existing ESP32 Olimex proxies if possible

**Comment by thaynes43** [comment]
I am interested in the answer to this question. I don't see a way to configure a Bluetooth proxy here.
